<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>batchWindow</class>
 <widget class="QDialog" name="batchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>375</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Batch download</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/resources.qrc">
    <normaloff>:/images/icon.ico</normaloff>:/images/icon.ico</iconset>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0">
   <item row="3" column="0" colspan="3">
    <widget class="QWidget" name="details" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>20</number>
        </attribute>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Batch</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Url</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Filesize</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Speed</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Progress</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="checkScrollToDownload">
          <property name="text">
           <string>Follow downloaded images</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonCopyToClipboard">
          <property name="text">
           <string>Copy links to clipboard</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>When the download is finished</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboEnd">
          <item>
           <property name="text">
            <string>Do nothing</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Close window</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Open CD tray</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Open destination folder</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Play a sound</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Shutdown</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkRemove">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="buttonDetails">
     <property name="text">
      <string>Details</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,0,0,0">
     <item>
      <widget class="QLabel" name="labelImages">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSpeed">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonPause">
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSkip">
       <property name="text">
        <string>Skip</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="labelMessage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QProgressBar" name="progressCurrent">
     <property name="value">
      <number>0</number>
     </property>
     <property name="format">
      <string notr="true">%p%</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QProgressBar" name="progressTotal">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonDetails</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>comboEnd</tabstop>
  <tabstop>checkRemove</tabstop>
  <tabstop>buttonPause</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonCopyToClipboard</sender>
   <signal>clicked()</signal>
   <receiver>batchWindow</receiver>
   <slot>copyToClipboard()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>batchWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonPause</sender>
   <signal>clicked()</signal>
   <receiver>batchWindow</receiver>
   <slot>pause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSkip</sender>
   <signal>clicked()</signal>
   <receiver>batchWindow</receiver>
   <slot>skip()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>copyToClipboard()</slot>
  <slot>pause()</slot>
  <slot>skip()</slot>
 </slots>
</ui>
